index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Etch A Sketch</title>
+
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <h1>Etch A Sketch</h1>
+    <h2>Swirl around and have fun</h2>
+    <div class="container">
+      <div class="controls">
+        <div class="controls-inner">
+          <h3>Controls</h3>
+          <div class="grid-size">8x8</div>
+          <input type="range" min="2" max="30" value="8" />
+          <button class="apply">Apply</button>
+          <button class="reset"></button>
+        </div>
+      </div>
+      <div class="grid"></div>
+    </div>
+
+    <script src="index.js"></script>
```

Comment:
We can't move script into the head section because when we run index.js it will require the class or id for dom manipulation and at that point of time no class or id will be found.

Reply:
where I have to do self closing tag?

Reply:
> self closing tag as well

Done

Reply:
> We can't move script into the head section because when we run index.js it will require the class or id for dom manipulation and at that point of time no class or id will be found.


Try using attributes like `async` and `defer` on the script tag ?

Reply:
> > We can't move script into the head section because when we run index.js it will require the class or id for dom manipulation and at that point of time no class or id will be found.
> 
> Try using attributes like `async` and `defer` on the script tag ?

It's working after adding defer